‏‏milestone-5/Fibonacci.js

```diff
@@ -0,0 +1,62 @@
+/* 
+ * ITC Full-Stack Bootcamp
+ * Fibonacci assignment
+ * 19/06/2022
+ * Asaf Gilboa
+ */
+
+
+async function fiboFetch(fiboServer) {
+
+  const response = await fetch(fiboServer);
+  if (!response.ok) {
+    const err = await response.text();
+    throw Error(err);
+  }
+  const fibo = await response.json();
+  return fibo.result;
+}
+
+async function fiboServerCall(num) {
+  let fiboServer = "http://localhost:5050/fibonacci/";
```

Comment:
Make it as a const, move it to higher position in the file, and use ALL_CAPS_FOR_CONSTANTS